lib/schnorr.py

```diff
@@ -130,6 +130,9 @@ def sign(privkey, message_hash):
     `message_hash` should be the 32 byte sha256d hash of the tx input (or
     message) you want to sign
     '''
+   
+    if not ndata is None:
```

Comment:
Pythonic tip: Prefer `a is not b` to `not a is b` -- even though they are both 100% equivalent the former is considered proper style.

Reply:
fixed in 35e621f